react/features/toolbox/components/Notice.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+
+import React, { Component } from 'react';
+
+import { translate } from '../../base/i18n';
+
+declare var config: Object;
```

Comment:
A preferred way would be to obtain this value from the `feature/base/config` instead of a global. Try searching how other components are doing "mapStateToProps"

Reply:
just sent a change about this